flow/workflows/cdc_flow.go

```diff
@@ -224,7 +321,9 @@ func CDCFlowWorkflowWithConfig(
 		if err := setupFlowFuture.Get(setupFlowCtx, &cfg); err != nil {
 			return state, fmt.Errorf("failed to execute child workflow: %w", err)
 		}
-		state.CurrentFlowState = protos.FlowStatus_STATUS_SNAPSHOT
+		state.SrcTableIdNameMapping = cfg.SrcTableIdNameMapping
```

Comment:
lets remove these from Config, its making it hard to reason about. It's ok for a breaking change.

Reply:
done as part of #1087